documentation/source/newsfragments/1632.change.rst

```diff
@@ -0,0 +1,2 @@
+``Makefile.pylib`` has been removed. 
+In case used ``PYTHON_LIBDIR`` or ``PYTHON_INCLUDEDIR`` have to be set by the user (see endian_swapper example).
```

Comment:
```suggestion
In case ``PYTHON_LIBDIR`` or ``PYTHON_INCLUDEDIR`` have been used, they now have to be set by the user (see the ``endian_swapper`` example).
```
(if that's what you wanted to say?)